fast_barcode_scanner/lib/src/overlays/code_boundary_overlay/code_border_painter.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+
+import '../../../fast_barcode_scanner.dart';
+
+class CodeBoarderPainter extends CustomPainter {
+  final CodeBorderPaintBuilder? barcodePaintSelector;
+  final CodeValueDisplayBuilder? textDecorator;
+
+  CodeBoarderPainter({
+    required this.imageSize,
+    required this.barcodes,
+    this.barcodePaintSelector,
+    this.textDecorator,
+  });
+
+  final Size imageSize;
+  final List<Barcode> barcodes;
+
+  final _standardPaint = Paint()
```

Comment:
Might this be regenerated for each Instance? I am unsure if the Painter object is recreated over and over, but `shouldRepaint` indicates just that.
Maybe make this static.